### ISSUE TITLE
Recommended parameters

### ISSUE THREAD

**Issue by PascalTemel** [7eu7d7/DreamArtist-sd-webui-extension#6]
It would be nice to have a "guide" on configuring the training params to get the same results as the previews in this repository.

For example:
* Number of vectors per token
* Number of negative vectors per token
* Initialization text (assuming I'm trying to recreate the nahida example, would I just use "girl"? Or "girl, white hair, dress, side ponytail" or something similar?)
* Learning rate
* CFG Scale
* Classifier (I'm using this https://github.com/7eu7d7/pixiv_AI_crawler/releases/tag/v2, is this correct?)
* Prompt template
* Max steps (how long do we have to train to see a decent result?)
* Any other options in the webui like "Stop At last layers of CLIP model"

Thanks for the work on this extension!

**Comment by princessGasmask** [comment]
Yeah, having SOMETHING to go off of would be great. I'm trying to train an artist style and I can't even tell if it's going well or not.

**Comment by 7eu7d7** [comment]
Add some pre-trained embeddings and its parameters to README, you can reference these parameters.
Use "girl" as initialization text is enough.
DreamArtist can be learned well without relying on filewords.

Classifier is mainly used for knowledge transfer, so if you don't use a lot of your own images for training, you may not use it. Unless your favorite style is similar to mine.

```Stop At last layers of CLIP model``` works well for both 1 and 2, it is hard to say which is better.

**Comment by princessGasmask** [comment]
Question on this, would you say it's "better" to use one image for training an artist style, or many images?

**Comment by henryvii99** [comment]
> Add some pre-trained embeddings and its parameters to README, you can reference these parameters. Use "girl" as initialization text is enough. DreamArtist can be learned well without relying on filewords.
> 
> Classifier is mainly used for knowledge transfer, so if you don't use a lot of your own images for training, you may not use it. Unless your favorite style is similar to mine.
> 
> `Stop At last layers of CLIP model` works well for both 1 and 2, it is hard to say which is better.

May I confirm that prompts typed in txt2img is only for preview purpose, and will not interfere training procedure?
If I want to intervene training by using keywords, I should incorporate in the filename, is that correct?

**Comment by yi03** [comment]
> Add some pre-trained embeddings and its parameters to README, you can reference these parameters. Use "girl" as initialization text is enough. DreamArtist can be learned well without relying on filewords.
> 
> Classifier is mainly used for knowledge transfer, so if you don't use a lot of your own images for training, you may not use it. Unless your favorite style is similar to mine.
> 
> `Stop At last layers of CLIP model` works well for both 1 and 2, it is hard to say which is better.

If I want to train a style, how should I fill in the initialization text? Is it okay to still fill in girl?
